### PR TITLE
Improve Italian translation

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Provided by @n00neheregit

### CODE REVIEW

File: Northstar.Client/mod/resource/northstar_client_localisation_italian.txt [hunk 1]
```diff
@@ -156,12 +156,12 @@ Premi Sì se sei d'accordo. Questa scelta può essere modificata in qualsiasi mo
 		"INFECTION_YOU_ARE_LAST_SURVIVOR" "Sei l'Ultimo Sopravvissuto!"
 		"INFECTION_SURVIVE_LAST_SURVIVOR" "Sopravvivi."
 
-		"PL_tffa", "Titan Free for All"
-		"PL_tffa_lobby", "Titan Free for All Lobby"
+		"PL_tffa", "Titan Tutti contro Tutti"
+		"PL_tffa_lobby", "Titan Tutti contro Tutti Lobby"
```
Comment (CloudSE7EN) [inline Northstar.Client/mod/resource/northstar_client_localisation_italian.txt:160]: line 160 can be changed in the same forms as the other "lobby" lines, changing it to "Lobby: Titan Tutti contro Tutti"

File: Northstar.Client/mod/resource/northstar_client_localisation_italian.txt [hunk 2]
```diff
@@ -126,11 +126,11 @@ Premi Sì se sei d'accordo. Questa scelta può essere modificata in qualsiasi mo
 
 		"PL_sns", "Sticks and Stones"
 		"PL_sns_lobby", "Sticks and Stones Lobby"
-		"PL_sns_desc", "Tutti contro Tutti, Usa la Pulse Blade e Esecuzioni per Resettare il Puntegio Nemico"
+		"PL_sns_desc", "Tutti contro Tutti, Usa la Pulse Blade e le Esecuzioni per Resettare il Puntegio Nemico"
 		"PL_sns_abbr", "SNS"
 		"GAMEMODE_SNS", "Sticks and Stones"
 		"SCOREBOARD_BANKRUPTS", "Uccisioni Bancarotta"
-		"SNS_LEADER_BANKRUPT", "Leader Punteggio andato in Bancarotta!"
+		"SNS_LEADER_BANKRUPT", "Il leader Punteggio andato in Bancarotta!"
```
[the remark this answers is not in the document]
Comment (GeckoEidechse) [reply Northstar.Client/mod/resource/northstar_client_localisation_italian.txt]: Done in 23c88df25b85804e626c8cf22a671b2bd25bfe92

File: Northstar.Client/mod/resource/northstar_client_localisation_italian.txt (hunk 1 above)
Comment (GeckoEidechse) [reply Northstar.Client/mod/resource/northstar_client_localisation_italian.txt]: Done in c7bfeb753b758b9366a90f40e7a66aaca6750f84

**Comment by GeckoEidechse** [comment]
> Changes made were confirmed by Aaron.#9925 in a private discord.
> 
> Also suggested 2 other lines image

The two suggested lines were updated in 5453486ed7456c087b680fe11caca36d6e9e9bb5 and 9cf9b0e94be565d28d3d90ff91c09d8cfa34ed60

**Pull request merged**